components/turret.py

```diff
@@ -72,42 +72,54 @@ def at_positive_limit(self) -> bool:
     def at_negative_limit(self) -> bool:
         return not self.negative_limit_switch.get()
 
+    @feedback
+    def has_found_index(self) -> bool:
+        return self.index_found
```

Comment:
Instead of adding a feedback method for this, how about making index_found a tunable?